README.md

```diff
@@ -32,12 +32,13 @@ You'll need to then configure the yaml yourself - the `type` part is filled out
 Here's an example yaml configuration: 
 
 ```
-entity: sensor.octopus_energy_electricity_<your_id_here_current_rate
+entity: sensor.octopus_energy_electricity_<your_id_here>_current_rate
 type: custom:octopus-energy-rates-card
 cols: 2
 showday: true
 showpast: false
 ```
+<your_id_here> consists of your "meter serial number" _ "meter MPAN number" it can be found in your Octopus account at https://octopus.energy/dashboard/developer/ eg. sensor.octopus_energy_electricity_12a3456789_2000001234567_current_rate
```

Comment:
I like this, but shall we explain to users how they can copy and paste it from the HA UI itself? I meant to make a note about how to go and get that *before* you start editing the card. 
e.g., "Hit search in top right -> search for `current_rate` ->  click the result -> click Settings tab -> copy `Entity ID`

Reply:
Yes, I agree I like that better. I'm forgetting that they would already have the entity installed, from the integration. There is already a similar explanation in the readme for that, (sensor.octopus_energy_electricity_{{METER_SERIAL_NUMBER}}_{{MPAN_NUMBER}}_current_rate )  so this card should definitely not repeat but refer to entity.